modules/contact/include/actions/ExplicitDynamicsContactAction.h

```diff
@@ -55,6 +55,9 @@ class ExplicitDynamicsContactAction : public Action
   /// Type that we use in Actions for declaring coupling
   typedef std::vector<VariableName> CoupledName;
 
+  /// Whether to run in an optimized form.
+  const bool _verbose;
```

Comment:
This doxygen string doesn't seem to pair well with the member name. I would think more verbose equals less optimized?